From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Refrag <R3FR4GYT@gmail.com>
Date: Sun, 4 Feb 2024 17:45:14 +0100
Subject: [PATCH] Add OpenMPT to build

The build system doesn't give us much choice on what library to build and what library to look for.
OpenMPT is not built by default but we need it to build so this goes and change that.
---
 CMakeLists.txt            | 1 -
 thirdparty/CMakeLists.txt | 2 +-
 2 files changed, 1 insertion(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 80a3bdcd6..9bf42970b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -130,7 +130,6 @@ if("${SRB2_CONFIG_SYSTEM_LIBRARIES}")
 	find_package(SDL2 REQUIRED)
 	find_package(SDL2_mixer REQUIRED)
 	find_package(CURL REQUIRED)
-	find_package(OPENMPT REQUIRED)
 
 	# libgme defaults to "Nuked" YM2612 emulator, which is
 	# very SLOW. The system library probably uses the
diff --git a/thirdparty/CMakeLists.txt b/thirdparty/CMakeLists.txt
index f33b3bf3f..141114e70 100644
--- a/thirdparty/CMakeLists.txt
+++ b/thirdparty/CMakeLists.txt
@@ -15,7 +15,7 @@ if(NOT "${SRB2_CONFIG_SYSTEM_LIBRARIES}")
 	include("cpm-zlib.cmake")
 	include("cpm-png.cmake")
 	include("cpm-curl.cmake")
-	include("cpm-openmpt.cmake")
 endif()
 
+include("cpm-openmpt.cmake")
 include("cpm-libgme.cmake")
